source/transactions-convenient-api/tests/README.md

```diff
@@ -41,8 +41,37 @@ If possible, drivers should implement these tests without requiring the test run
 the retry timeout. This might be done by internally modifying the timeout value used by `withTransaction` with some
 private API or using a mock timer.
 
+### Retry Backoff is Enforced
+
+Drivers should test that retries within `withTransaction` do not occur immediately. First, run transactions without
+backoff. To do so, configure the random number generator used for jitter to always return `0` -- this effectively
+disables backoff. Then, configure a fail point that forces 30 retries like so:
+
+```json
+{
+    "configureFailPoint": "failCommand",
+    "mode": {
+        "times": 13
+    },
+    "data": {
+        "failCommands": ["commitTransaction"],
+        "errorCode": 251,  // NoSuchTransaction
+    },
+}
+```
+
+Let the callback for the transaction be a simple `insertOne` command. Let `no_backoff_time` be the time it took for the
+command to succeed.
```

Comment:
suggest clarifying here what we mean "time it took for the command to succeed".  I think this means "measure the duration of the withTransaction API call"?